heareval/tasks/config/nsynth_pitch.py

```diff
@@ -0,0 +1,24 @@
+"""
+Configuration for the nsynth pitch detection task
+"""
+
+from .dataset_config import *  # noqa: F403, F401
+
+TASKNAME = "nsynth-pitch-v2.3.3"
+
```

Comment:
Weird there's no version number here

Reply:
Ya let's add that as a separate config var?